src/test/java/pageobject_model/WebDriverSeleniumPageDataValidationTest.java

```diff
@@ -1,18 +1,28 @@
-package hurtmeplentytest;
+package pageobject_model;
 
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
 import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
 import org.testng.annotations.Test;
+import pageobject_model.page.CloudGoogleCalculatorPage;
 
 public class WebDriverSeleniumPageDataValidationTest extends BaseTest {
 
-    @Test
-    public void testGetCorrectEstimationValue() {
-        Assert.assertEquals("Total Estimated Cost: USD 1,187.77 per 1 month", BaseTest.getEstimateResult());
+    @BeforeTest(alwaysRun = true)
+    public static void setUp() {
+        WebDriver driver = new ChromeDriver();
+        driver.manage().window().maximize();
     }
 
     @Test
-    public void testGetVmClassCorrectValue() {
-        Assert.assertEquals("VM class: regular", BaseTest.getVmClass());
+    public static void testGetCorrectEstimationValue() {
+        String estimateResult = new CloudGoogleCalculatorPage()
+                .openCalulatorPage()
+                .fillingFor
+                .gettingEstimateReults;
+        Assert.assertEquals("Total Estimated Cost: USD 1,187.77 per 1 month", BaseTest.getEstimateResult());
```

Comment:
1) it's always better to have comment for assertion(ex: assertEqual(message, expected, actual));
2) It's really bad idea to compare whole string. If you are checking price - you should write assertion only for price field and it will be better if this price will be in digits (not like a string);
3) I didn't get what is idea of such syntaxes: BaseTest.getEstimateResult();